sycl/test-e2e/DeviceLib/exp/exp-std-complex-edge-cases.hpp

```diff
@@ -1,17 +1,10 @@
 // This test checks edge cases handling for std::exp(std::complex<T>) used
 // in SYCL kernels.
-//
-// REQUIRES: aspect-fp64
```

Comment:
Note for other reviewers who may also think that this is an incorrect change: this is an `.hpp` file that is used from `.cpp` file which is the actual test, so it is fine.